test/strongbox/core_test.clj

```diff
@@ -593,18 +593,45 @@
                           :source "curseforge",
                           :source-id 1}]
         (is result) ;; success
-        (is (= ["BundledAddon" "EveryAddon"] directory-list))
-        (is (= expected-nfo (nfo/read-nfo-file install-dir "BundledAddon")))))))
+        (is (= ["EveryAddon" "EveryAddon-BundledAddon"] directory-list))
+        (is (= expected-nfo (nfo/read-nfo-file install-dir "EveryAddon-BundledAddon")))))))
+
+(deftest install-bundled-addon-overwriting-ignored-addon
+  (testing "installing/unzipping an addon with a shared mutual dependency of an addon that is ignored isn't possible"
+    (with-running-app
+      (let [install-dir (helper/addons-path)
+            _ (core/set-addon-dir! install-dir)
+            install-dir-contents #(->> install-dir fs/list-dir (map fs/base-name) sort)
```

Comment:
I use this often enough it should probably join help/addons-path in test_helper.clj